src/User32.Desktop/User32.cs

```diff
@@ -13,6 +13,8 @@ namespace PInvoke
     [OfferFriendlyOverloads]
     public static partial class User32
     {
+        public const int WHEEL_DELTA = 120;
```

Comment:
I see. That sounds good then, if you could just add that description to an xml doc comment that would be great.